python/pypto/language/arg_direction.py

```
@@ -0,0 +1,119 @@
+
+__all__ = [
+    "ArgDirection",
+    "DIRECTION_TO_NAME",
+    "NAME_TO_DIRECTION",
+    "input",
+    "inout",
+    "no_dep",
+    "output",
+    "output_existing",
+    "scalar",
+]
```

Ruff (0.15.10)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)